mix.exs

```diff
@@ -40,15 +41,15 @@ defmodule Blockchain.Mixfile do
   # Type "mix help deps" for more examples and options
   defp deps do
     [
-      {:libsecp256k1, "~> 0.1.2"},
+      {:libsecp256k1, github: "omisego/libsecp256k1"},
```

Comment:
You cannot publish a package to Hex with a github dep

Reply:
yep. I wait for libsecp256k1 update.

Reply:
Here ya go! https://hex.pm/packages/libsecp256k1/0.1.4